### ISSUE TITLE
Getting an error while Start the frontend

### ISSUE THREAD

**Issue by siva2sachin** [cdoremus/fileupload-demo-angular2#1]
Getting below error while running NPM start,

![image](https://cloud.githubusercontent.com/assets/10898928/23342216/151de564-fc1c-11e6-9cb3-45a17ec05915.png)

Could you please look into this issue. 

![image](https://cloud.githubusercontent.com/assets/10898928/23342254/9a4e3efa-fc1c-11e6-8b09-4fed1d0e7b5b.png)

**Comment by cdoremus** [comment]
I haven't worked on this in a few months, but I can replicate it this issue. This repo is based on the angular2-webpack repo from the github preboot user. I checked with their Gitter discussion board and others are having the same issue. 

I was able to fix this by moving my src folder files into a new angular2-webpack cloned repo and changing port 8080 to 8000 in package.json, webpack.config.js and karma.conf.js. You might want to try  to do the same as I am unsure when I'll be able to fix this issue.
